snapshot-labs/snapshot
# feat(unsubscribe): add checkbox for removing email from DB

Fixes #
Adds the ability to remove email from DB altogether.

### How to test
1. Login with subscribed account
2. Open manage email subscription modal
3. Turn off all options
4. The checkbox should appear
5. Turn on the checkbox and update your preferences
6. Ensure that you are no longer subscribed for notifications and for subscription you need to enter your email again and sign a request to BE
<img width="791" alt="Screenshot 2023-06-09 at 10 51 45" src="https://github.com/snapshot-labs/snapshot/assets/7693369/bbdcf901-9ee2-4040-9cc5-cec59c621746">


### To-Do


### Self-review checklist
